contrib/kubespray/roles/docker-cache/install/defaults/main.yml

```diff
@@ -0,0 +1 @@
+enable_docker_cache: false
```

Comment:
add new line in the end of file, you could use editorconfig plugin in vscode for auto processing